hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -59,15 +59,18 @@ export default function HackathonDetails(props: IProps) {
   const [hackathonData, setHackathonData] = useState({} as Hackathon)
```

Comment:
so I'm guessing this whole issues is stemming from this line here.  If you were to give description a value here it would fix it, so...
```suggestion
  const [hackathonData, setHackathonData] = useState({
    description: ''
  } as Hackathon)
```

In general all the places where you have been using these types might benefit from them being classes instead.  And then you could have default constructors that make a good "empty" instance of the classes.